README.md

```diff
@@ -16,11 +16,15 @@ gem "lita-giphy"
 ### Required attributes
 * `api_key` - Your [Giphy API](https://github.com/giphy/GiphyAPI) key. You can either email the devs for a personal API key, or you can use the default public beta key-- `dc6zaTOxFJmzC`.
 
+### Optional attributes
+* `rating` - allows you to set the rating of the gifs that are found
```

Comment:
This is great, but I think it would be most helpful if the README also included the allowed values of this `rating` attribute. That way, people won't have to go look up the Giphy API documentation to use this handler.